specification/common-types/resource-management/v5/types.json

```diff
@@ -635,28 +732,32 @@
       "description": "The API version to use for this operation.",
       "required": true,
       "type": "string",
-      "minLength": 1
+      "minLength": 1,
+      "x-ms-parameter-location": "client",
+      "x-ms-client-name": "apiVersion"
     },
     "If-Match": {
-      "name": "ifMatch",
+      "name": "If-Match",
       "in": "header",
       "description": "The If-Match header that makes a request conditional.",
       "required": true,
       "type": "string",
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "ifMatch"
     },
     "If-None-Match": {
-      "name": "ifNoneMatch",
+      "name": "If-None-Match",
       "in": "header",
       "description": "The If-None-Match header that makes a request conditional.",
       "required": true,
       "type": "string",
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "ifNoneMatch"
     },
     "LocationParameter": {
       "name": "location",
       "in": "path",
-      "description": "The name of the Azure region.",
+      "description": "The name of Azure region.",
```

Comment:
>Azure [](http://example.com/codeflow?start=34&length=5)

I think 'the Azure region' or 'an Azure region' made sense... if we imagine it to be shorthand for '[the/an] Azure region a resource is deployed in'